src/types/Metrics.ts

```diff
@@ -1,6 +1,6 @@
 export interface Metrics {
-  metrics: Map<String, MetricGroup>;
-  histograms: Map<String, Histogram>;
+  metrics: { [k: string]: MetricGroup };
```

Comment:
I would like to see a "key" instead of "k" but I know this is trivial.
For me "key" heps.